src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -630,6 +630,20 @@ def cancel_job(self, job_id: str, parent_job_id: str = None) -> None:
                 self._running_jobs[job_id]["refresh"] = is_refreshing
                 del self._running_jobs[job_id]["canceling"]
 
+    def retry_job(self, job_id: str):
+        if job_id is None:
+            raise ValueError("Job id required for retrying")
+        if job_id not in self._running_jobs:
```

Comment:
can you take out this check and just let ee2 send back the appropriate error message?